### DISCUSSION TITLE
Question regarding Printer for the MCDU Webserver

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#6436] [Q&A]
The new MCDU Webserver and Printer Feature is so awesome! Thank you for his huge improvement.

Is it possible to use a small terminal Printer for this?  like this one? 

https://www.amazon.de/NETUM-Bluetooth-Belegungs-Thermodrucker-pers%C3%B6nlicher-Rechnungsdrucker-Mini-USB-POS-Drucker/dp/B0854CCF75/ref=sr_1_10?__mk_de_DE=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=23W05XHQTJOPF&keywords=therminal+printer&qid=1640806737&sprefix=terminal+printe%2Caps%2C71&sr=8-10

thank you for an answer!

**Comment by plhought** [comment]
I attempted to use the print feature with a similar thermal printer and although it attempted to print - the text was very small and unreadable. I have already made a bug report.
